content/blog/eclipse-temurin-8u462-11028-17016-2108-2402-available/index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Eclipse Temurin 8u462, 11.0.28, 17.0.16, 21.0.8 and 24.0.2 Available
+date: "2025-07-22"
+author: pmc
+description: Adoptium is happy to announce the immediate availability of Eclipse Temurin 8u462, 11.0.28, 17.0.16, 21.0.8 and 24.0.2. As always, all binaries are thoroughly tested and available free of charge without usage restrictions on a wide range of platforms.
+tags:
+  - temurin
+  - announcement
+  - release-notes
+---
+
+Adoptium is happy to announce the immediate availability of Eclipse Temurin 8u462-b08, 11.0.28+6, 17.0.16+8, 21.0.8+9 and 24.0.2+12. As always, all binaries are thoroughly tested and available free of charge without usage restrictions on a wide range of platforms. Binaries, installers, and source code are available from the [Temurin download page](https://adoptium.net/temurin/releases), [official container images](https://hub.docker.com/_/eclipse-temurin) are available at DockerHub, and [installable packages](https://adoptium.net/installation/) are available for various operating systems.
+
+## Fixes and Updates
+
+This release contains the following fixes and updates.
+
+* [Temurin 8u462 release notes](https://adoptium.net/temurin/release-notes/?version=jdk8u462-b08)
+
+* [Temurin 11.0.28 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-11.0.28+6)
+
+* [Temurin 17.0.16 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-17.0.16+8)
+
+* [Temurin 21.0.8 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-21.0.8+9)
+
+* [Temurin 24.0.2 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-24.0.2+12)
+
+## New and Noteworthy
+
```

Comment:
We should call out https://github.com/adoptium/containers/pull/782

Specifically that we've introduced an Alpine 3.22 image which will now be the default Alpine version when someone pulls a tag such as `eclipse-temurin:21-alpine`. They will be able to stick to the previous version if they want by updating their Dockerfiles to pull from `eclipse-temurin:21-alpine-3.21`